pkg/security/resolvers/time/resolver.go

```diff
@@ -63,8 +63,12 @@ func (tr *Resolver) ApplyBootTime(timestamp time.Time) time.Time {
 // ComputeMonotonicTimestamp converts an absolute time to a kernel monotonic timestamp
 func (tr *Resolver) ComputeMonotonicTimestamp(timestamp time.Time) int64 {
 	if !timestamp.IsZero() {
-		offset := tr.getUptimeOffset()
```

Comment:
Discussed offline, but just to make sure we don't forget: I think we need to add the uptime offset unfortunately :(

Reply:
Something I didn't mention during our discussion but the offset is already used in the GetBootTime function

Reply:
nice